<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssConv</class>
 <widget class="QWidget" name="AssConv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>430</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Ass Convertor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="labelFormat">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFormat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>26</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Ass2Lrc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ass2Txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lrc2Ass</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Txt2Ass</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Png2Ass</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonConvert">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/buttons/wand.png</normaloff>:/buttons/wand.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/buttons/cross_grey.png</normaloff>:/buttons/cross_grey.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>AssConv</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
